packages/core/src/tree-view/utils.ts

```diff
@@ -0,0 +1,50 @@
+export type CheckboxSelection = true | false | 'indeterminate';
```

Comment:
I didn't think it was worth adding a utils directory since there are only two of them.

Reply:
May want to add a comment block at the top of this file that these are for storybook demo utils so it does not accidentally get pulled into published library code. Maybe the file could be `story-utils.ts` or `demo.ts`?